repo/open.go

```diff
@@ -379,21 +387,21 @@ func readAndCacheFormatBlobBytes(ctx context.Context, st blob.Storage, cacheDire
 	if cacheEnabled {
 		b, err := readFormatBlobBytesFromCache(ctx, cachedFile, validDuration)
```

Comment:
Do these two functions need to be renamed? They aren't exclusively used for `formatBlob`'s anymore

Reply:
Yes, will rename them too.